packages/app/src/components/home/HomePage.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Content, Header, Lifecycle, Page } from '@backstage/core-components';
+import Grid from '@material-ui/core/Grid';
+import { RandomJokeHomePageComponent } from '@backstage/plugin-home';
+
+export const HomePage = () => {
+  return (
+    <Page themeId="home">
+      <Header title="Home" subtitle={<Lifecycle alpha />} />
+      <Content>
+        <Grid container spacing={3}>
+          <Grid item xs={12} md={4}>
+            <RandomJokeHomePageComponent />
```

Comment:
To play back my understanding as a potential plugin developer...

*  I could develop a plugin called @mycompany/agile-tool.
* I might develop cards, but those cards in may plugin would be exposed to individual component pages as normal.
* However, If I wanted to develop a homepage card for my plugin, I'd actually have to fork the main Backstage `@backstage/plugin-home` component, and develop my code inside of that, contribute it back to the main project, and have it merged in and released - is that correct?

If so, I feel like that blurs the boundary of my ability to release my plugin independently of the app itself.  It may just be how this specific example is used, since RandomJoke doesn't have it's own plugin, but wanted to check if I'm understanding it correctly?

Otherwise, I might hope to be able to develop a component inside of my `@mycompany/agile-tool` plugin, and extend a shared "homepage card component" that I can then export out of my plugin, and then re-import it in my HomePage.tsx here?  So effectively, the plugin gives me that structure, but doesn't ITSELF expose my components?

Does that make sense?

Reply:
Yes, the second way you describe it is correct! 

If the the HomePage Component has no corresponding plugin (such as the RandomJoke), in which case it lives in the Home plugin

Otherwise, if the HomePage Component belongs to some plugin in some way, it's included there. This applies no matter if the plugin is part of the backstage/backstage monorepo or not. So for the @mycompany/agile-tool, you can develop a component within your own plugin that is then exposed as an export. To do this you would supposedly only need the `createHomePageComponentExtension` from the HomePage plugin (the way it's designed right now, at least). 

So yes, makes sense, and I think we're on the same page! :rocket: